spyder/plugins/updatemanager/workers.py

```diff
@@ -51,6 +51,12 @@
     '<br><br>Please contact your network administrator for assistance.'
 )
 
+OS_ERROR_MSG = _(
+    "An error occurred while checking for Spyder updates, possibly related to "
+    "your operating system configuration or file access.<br><br>If you're not "
+    "sure what to do about it, you can disable checking for updates below. "
+    "<br><br>The error was:<br><br><tt>{error}</tt>"
```

Comment:
```suggestion
    "<br><br>The error was:<br><br><i>{error}</i>"
```

I just noticed that we use italics to show error messages in the Variable Explorer, so I think we should to the same here.